tests/std/tests/P0896R4_views_reverse/test.cpp

```diff
@@ -317,6 +317,13 @@ using move_only_view = test::range<Category, const int, test::Sized{is_random},
     IsCommon, test::CanCompare{derived_from<Category, forward_iterator_tag>},
     test::ProxyRef{!derived_from<Category, contiguous_iterator_tag>}, test::CanView::yes, test::Copyability::move_only>;
 
+void test_gh2312() {

```

Comment:
Let's remind our future selves that this code doesn't actually execute:
```suggestion
void test_gh2312() { // COMPILE-ONLY
```